terraform/etl/48-lambda-gov-notify-ingestion.tf

```
@@ -1,3 +1,9 @@
+locals {
+  govnotify_tables                = ["notifications", "received_text_messages"]
+  create_govnotify_resource_count = 1
```

assuming you don't want everything in prod for now you can set this to 
```suggestion
  create_govnotify_resource_count = !local.is_production_environment ? 1 :  0
```
and that should sort the issue with your prod plan